utils/lightningServices.js

```diff
@@ -41,21 +43,18 @@ class LightningServices {
     /**
      * @type {Config}
      */
-    const newDefaults = {
-      ...require("../config/defaults")(program.mainnet),
-      useTLS: false,
-    }
+    const newDefaults = require("../config/defaults")(program.mainnet)
 
     this.defaults = newDefaults;
 
     this._config = {
+      ...newDefaults,
       useTLS: program.usetls,
       serverPort: program.serverport || newDefaults.serverPort,
       serverHost: program.serverhost || newDefaults.serverHost,
       lndHost: program.lndhost || newDefaults.lndHost,
       lndCertPath: program.lndCertPath || newDefaults.lndCertPath,
-      macaroonPath: program.macaroonPath || newDefaults.macaroonPath,
-      lndProto: newDefaults.lndProto
```

Comment:
I don't follow

Reply:
lndProto is already defined in ./config/defaults so there's no need to re-specify the same thing here if we're spreading the newDefaults object